frontend/src/pages/forms/Form.jsx

```diff
@@ -63,52 +52,48 @@ const Form = ({ walletId, setWalletId }) => {
     }
     return null;
   };
+  const checkOpenPosition = () => {
+    // write api check for position
+    return true;
+  };
```

Comment:
finish this, seems like you could use pervious API call and based on that either proceed or show position modal?